pkg/venafi/tpp/connector.go

```diff
@@ -582,6 +582,21 @@ func prepareRequest(req *certificate.Request, zone string) (tppReq certificateRe
 		tppReq.CASpecificAttributes = append(tppReq.CASpecificAttributes, nameValuePair{Name: expirationDateAttribute, Value: formattedExpirationDate})
 	}
 
+	if req.ValidityDuration != nil {
+		for _, attributeName := range []string{
+			"Microsoft CA:Specific End Date",
+			"DigiCert CA:Specific End Date",
+			"EntrustNET CA:Specific End Date",
+			"Specific End Date",
+		} {
```

Comment:
instead of using an IssuerHint paramter, we just add the `Specific End Date` option for all the issuers that we know.

Reply:
The documentation shows that there are more than those three issuers:
 * https://docs.venafi.com/Docs/currentSDK/TopNav/Content/SDK/WebSDK/SchemaReference/r-SDK-CA-SpecificAttributes.php

And some of them (e.g. Sectigo) require a validity duration rather than a specific end date.

![image](https://github.com/jetstack/vcert/assets/978965/4b90accd-b414-466e-a9a8-87534de85ae3)

-- https://docs.venafi.com/Docs/currentSDK/TopNav/Content/SDK/WebSDK/SchemaReference/r-SDK-CA-SectigoFormerlyComodoCCM.php

I guess we can add support for those in followup PRs?